### ISSUE TITLE
Saucelabs tests aren't running but still passing CI

### ISSUE THREAD

**Issue by paulfalgout** [marionettejs/backbone.marionette#3343]
https://travis-ci.org/marionettejs/backbone.marionette/jobs/221224722
```
(node:2708) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): Error: A Sauce Labs username and access key are required.
```

@denar90 new key needed or something?

**Comment by paulfalgout** [comment]
I don't know if this is really broken. It worked on the test of the most recent merge

**Comment by denar90** [comment]
@paulfalgout looks like an issue. Running tests in browser are broken
Here's screen from soucelabs... 
I'll take a look why it's happening... 
![0004screenshot](https://cloud.githubusercontent.com/assets/6231516/24950287/4801c1a0-1f79-11e7-8f71-5b6eb3033eb4.png)

**Comment by paulfalgout** [comment]
I think there's 2 things going on here #3346 fixes `gulp test-browser` but https://travis-ci.org/marionettejs/backbone.marionette/jobs/221403339 is still not launching saucelabs.  I think it does though when something gets merged into develop

**Comment by denar90** [comment]
@paulfalgout have you faced same error about sinon during `gulp test-browser` locally?

**Comment by denar90** [comment]
Nice
![image](https://cloud.githubusercontent.com/assets/6231516/24981470/927043bc-1fe5-11e7-8910-0e843239c738.png)

solved by #3346

@paulfalgout cheers 🍺

**Comment by denar90** [comment]
Yeap, but another thing that for  $TEST_SUITE = "browser" it's ok, but for  $TEST_SUITE = "browser" with lodash is not...

I think because these 2 tests are run on the same port `8080` as result case with lodash which was run as second, gets this error
```
Started static web server on port 8080
(node:2689) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: 12 Apr 22:05:25 - Sauce Connect could not establish a connection.
```
because port already used, or closed, or smth...
